Add Vault guides (#345)
lukso-network/docs: docs/guides/vault/restrict-addresses-to-vaults.md

---
sidebar_label: 'Restrict Addresses to Vaults'
sidebar_position: 4
---

# Restrict Addresses to Vaults

> As mentioned in the [first Vault guide](./create-a-vault.md), the **Vault** can be used to restrict different addresses (protocols, other devices, etc..) to execute and set data on it instead of setting and executing directly on the Universal Profile. 

Reviewer: ```suggestion
As mentioned in the [first Vault guide](./create-a-vault.md), the **Vault** can be used to restrict different addresses (protocols, other devices, etc.) to execute and set data on it instead of doing it directly on the Universal Profile. 
```

> In this way, when **granting a third party permissions** to execute through your profile, this third party will only be able to interact with the Vault and all the other assets will be safe.

Reviewer: ```suggestion
This way, when **granting a third party permissions** to execute through your profile, this third party will only be able to interact with the Vault, and all the other assets will be safe.
```

![Guide - Restrict addresses to an LSP9Vault](/img/guides/restrict-protocol-to-vault.jpeg)

## Granting Permission to 3rd Parties

:::note

> Make sure to not grant the 3rd party address the **SUPER Permissions**, otherwise the **AllowedAddresses restriction** will not work.

Reviewer: ```suggestion
Make sure not to grant the 3rd party address the **SUPER Permissions**. Otherwise, the **AllowedAddresses restriction** will not work.
```
